### PR TITLE
Rrefactor(components/molecule/badgeCounter): Refactor style to badgeCunter with three characters

### PR DESCRIPTION
**Pull request by IsabelGomez87**
## Category/Component
components/molecule/badgeCounter
 #### `❓ Ask` 

**TASK**: none

### Types of changes

- [ ] 🪲 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [x] 💄 Styles

### Description, Motivation and Context
Refactor style to badgeCunter with three characters to accomplish with the definition of UX styles

### CODE REVIEW

File: components/molecule/badgeCounter/src/styles/settings.scss [hunk 1]
```diff
@@ -30,6 +30,10 @@ $molecule-badge-counter-large-offset-right: -($sz-base + $sz-base * 0.5) !defaul
 $fz-molecule-badge-counter-large: $fz-xs !default;
 $p-molecule-badge-counter-large: 0 $p-base !default;
 
+/* threeCharacters */
+$bdrs-molecule-badge-counter-threeCharacters: $bdrs-xxxl !default;
+$molecule-badge-counter-threeCharacters-diameter: 36px !default;
```
Comment (andresin87) [inline components/molecule/badgeCounter/src/styles/settings.scss:35]: haave you think about teh possibilityb to make it dependant to a `ch` unit?

Comment (andresin87) [reply components/molecule/badgeCounter/src/styles/settings.scss:35]: https://www.w3schools.com/cssref/css_units.asp

Comment (andresin87) [reply components/molecule/badgeCounter/src/styles/settings.scss:35]: https://caniuse.com/ch-unit

Comment (andresin87) [reply components/molecule/badgeCounter/src/styles/settings.scss:35]: Not available for IE11 😭

Comment (andresin87) [reply components/molecule/badgeCounter/src/styles/settings.scss:35]: IE supports the ch unit, but unlike other browsers its width is that specifically of the "0" glyph, not its surrounding space. As a result, 3ch for example is shorter than the width of the string "000" in IE.

Comment (turolopezsanabria) [reply components/molecule/badgeCounter/src/styles/settings.scss:35]: Do we still support IE 11?

File: components/molecule/badgeCounter/src/styles/settings.scss [hunk 2]
```diff
@@ -30,6 +30,10 @@ $molecule-badge-counter-large-offset-right: -($sz-base + $sz-base * 0.5) !defaul
 $fz-molecule-badge-counter-large: $fz-xs !default;
 $p-molecule-badge-counter-large: 0 $p-base !default;
 
+/* threeCharacters */
+$bdrs-molecule-badge-counter-threeCharacters: $bdrs-xxxl !default;
```
Comment (turolopezsanabria) [inline components/molecule/badgeCounter/src/styles/settings.scss:34]:
```suggestion
$bdrs-molecule-badge-counter-threeCharacters: $bdrs-molecule-badge-counter !default;
```
it has to be like this for retro-compatibility and later replace it to `$bdrs-xxxl` on your theme

**Review by turolopezsanabria — CHANGES_REQUESTED** [review]

File: components/molecule/badgeCounter/src/styles/settings.scss (hunk 2 above)
Comment (IsabelGomez87) [reply components/molecule/badgeCounter/src/styles/settings.scss:34]: But, `$bdrs-molecule-badge-counter` is `$bdrs-xxl` (16px) and for `$bdrs-molecule-badge-counter-threeCharacters` it's necessary `$bdrs-xxl` (32px), by UX demand. For this reason, I have declared a new variable `$bdrs-molecule-badge-counter-threeCharacters`

**Review by andresin87 — APPROVED** [review]

**Review by turolopezsanabria — APPROVED** [review]
Thnx!

**Pull request merged**